biz.aQute.bndlib/src/aQute/bnd/differ/Baseline.java

```diff
@@ -204,13 +204,19 @@ public boolean contains(Diff diff) {
 					content = pdiff.getDelta();
 					break;
 
-				case REMOVED :
+				case REMOVED : // this is a package removal not a member removal
```

Comment:
This change is altering REMOVED from MAJOR to MICRO which is a big reduction. It is possible the bug in the compareTo then clause is the reason changes were lost.